convert2rhel/actions/report.py

```diff
@@ -75,9 +76,13 @@ def summary_as_json(results, json_file=CONVERT2RHEL_JSON_RESULTS):
     * The results are modified so that status codes use their symbolic names
       instead of the numeric values.
     """
+    # Collect the highest report level to use as status key
+    highest_level = find_highest_report_level(results)
+
     # Use an envelope so we can add other, non-result info if necessary.
     envelope = {
         "format_version": "1.0",
+        "status": highest_level,
```

Comment:
The `schemas/assessment-schema-1.0.json` is missing the `status` item

Reply:
Thanks i'll add that